recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
-{% set version = "16.0.6" %}
+{% set version = "17.0.2" %}
 
 package:
   name: lldb
   version: {{ version }}
 
 source:
   url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version }}/llvm-project-{{ version }}.src.tar.xz
-  sha256: ce5e71081d17ce9e86d7cbcfa28c4b04b9300f8fb7e78422b1feb6bc52c3028e
+  sha256: 351562b14d42fcefcbf00cc1f327680a1062bbbf67a1e1ca6acb64c473b06394
   patches:
     # Fix Python and Mac support
     - patches/0001-Remove-linking-liblldb-to-libpython.patch  # [not win]
     # https://github.com/llvm/llvm-project/issues/59449
     # allows for compiling with older SDKs
     - patches/0001-Define-CPU_SUBTYPE_ARM64E-if-not-available.patch
-    # https://reviews.llvm.org/D156763
-    - patches/6888de118707e6392b46073fc35738804f9f1d80.patch
 
 build:
   number: 0
+  # This needs clang-compiler-activation-feedstock build
+  skip: true  # [osx and arm64]
```

Comment:
If you're fine with that, so I am I. We could also unblock clang 17 if we loosen the stdlib-dep to `libcxx >=16`.

Reply:
That would be my preferred approach, but I have no idea how this would work. I still haven't understood the `stdlib` concept :(